crates/chain/src/tx_graph.rs

```diff
@@ -683,7 +695,7 @@ impl<A: Anchor> TxGraph<A> {
 
         // If a conflicting tx is in the best chain, or has `last_seen` higher than this tx, then
         // this tx cannot exist in the best chain
-        for conflicting_tx in self.walk_conflicts(tx, |_, txid| self.get_tx_node(txid)) {
+        for conflicting_tx in self.walk_conflicts(tx, 25, |_, txid| self.get_tx_node(txid)) {
```

Comment:
I think it would make sense to make the `depth_limit` input configurable.

However, because `try_get_chain_position` is called in so many places, would it make sense to make all these methods take in an `ancestor_depth_limit` argument?

The other option is to have the `ancestor_depth_limit` global for the entire `TxGraph`, but would there be a situation where we want different depth limits for the same graph?

Is there even a way to avoid the depth limit entirely?

Reply:
I think as long as we accept this `25` is a temporary solution it's fine to leave it fixed. Let's put it in a constant.